payloads/v8/_interactions/autocomplete.ts

```diff
@@ -0,0 +1,25 @@
+import type { APIBaseInteraction, ApplicationCommandOptionType, InteractionType } from '..';
```

Comment:
Please fix this import so it points directly to the file rather than to `..` (for instance `../index` or w/e)